services/subnet_manager/src/main/java/com/futurewei/alcor/subnet/dao/SubnetRepository.java

```diff
@@ -33,21 +34,21 @@
 
 @Repository
 @ComponentScan(value="com.futurewei.alcor.common.db")
-public class SubnetRepository implements ICacheRepository<SubnetWebObject> {
+public class SubnetRepository implements ICacheRepository<SubnetWebResponseObject> {
```

Comment:
@kevin-zhonghao I am thinking of giving this class "SubnetWebResponseObject" a better name, like SubnetWebResponseEntity or something else? Any Suggestion?

Reply:
Actually SubnetWebResponseEntity  is better than SubnetWebResponseObject, it is easier to understand. 
I think maybe we could just name it as SubnetEntity, because we store  SubnetWebResponseObject as subnet entity in db. What do you think

Reply:
Sounds good to me.

Reply:
NP